src/shared/components/entry/EntryDownload.tsx

```diff
@@ -279,46 +379,72 @@ const EntryDownload = ({
 
   const proteinsApiVariation = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.variation(accession))
+      ? apiUrls.proteinsApi.variation(accession)
       : '',
     { method: 'HEAD' }
   );
 
   const proteinsApiProteomics = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.proteomics(accession))
+      ? apiUrls.proteinsApi.proteomics(accession)
       : '',
     { method: 'HEAD' }
   );
 
   const proteinsApiPTMs = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.proteomicsPtm(accession))
+      ? apiUrls.proteinsApi.proteomicsPtm(accession)
       : '',
     { method: 'HEAD' }
   );
 
   const proteinsApiMutagenesis = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.mutagenesis(accession))
+      ? apiUrls.proteinsApi.mutagenesis(accession)
       : '',
     { method: 'HEAD' }
   );
 
   const proteinsApiAntigen = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.antigen(accession))
+      ? apiUrls.proteinsApi.antigen(accession)
       : '',
     { method: 'HEAD' }
   );
 
   const proteinsApiCoordinates = useDataApi(
     namespace === Namespace.uniprotkb && accession
-      ? joinUrl(apiUrls.proteinsApi.coordinates(accession))
+      ? apiUrls.proteinsApi.coordinates(accession)
+      : '',
+    { method: 'HEAD' }
+  );
+
+  const interProRepresentativeDomains = useDataApi(
+    (namespace === Namespace.uniprotkb || namespace === Namespace.uniparc) &&
+      accession
+      ? externalUrls.InterProRepresentativeDomains(accession)
       : '',
     { method: 'HEAD' }
   );
 
+  const alphaFoldPrediction = useDataApi<AlphafoldPayload>(
+    namespace === Namespace.uniprotkb && accession
+      ? externalUrls.AlphaFoldPrediction(accession)
+      : ''
+  );
+
+  const alphaFoldUrls =
+    // As there can be a build mismatch only use AlphaFold predictions if the sequence is the same as the entry's
+    sequence && sequence === alphaFoldPrediction?.data?.[0].uniprotSequence
+      ? getAlphaFoldUrls(alphaFoldPrediction?.data)
+      : undefined;
+
+  if (alphaFoldUrls) {
+    availableDatasets.push(Dataset.alphaFoldCoordinates);
+    availableDatasets.push(Dataset.alphaFoldConfidence);
+    availableDatasets.push(Dataset.alphaMissenseAnnotations);
```

Comment:
There's an example that someone from Ensembl found _with_ AF, but _without_ AM: https://alphafold.ebi.ac.uk/api/prediction/A0A2U3U0J3
Have a look if you could add an extra check for that. Not sure if it's expected or not, PDBe will look into it, but now we see that it _could_ happen

Reply:
That's a good example because it also has these other fields I'm not considering:
```
"amAnnotationsHg19Url": null,
"amAnnotationsHg38Url": null
```
Add these as options too you think?